src/clj/game/cards/programs.clj

```diff
@@ -2535,42 +2535,53 @@
                 (strength-pump 2 2)))
 
 (defcard "Pawn"
-  {:implementation "[Erratum] Program: Caïssa - Trojan. Developer Note: All abilities are manual"
```

Comment:
We should keep the errata implementation note because people seem to like those.